definitions/hackingai.json

```diff
@@ -0,0 +1,32 @@
+{
+
+    "mlflow": {
+        "process_name": ["python*.exe",
+                         "python*",
+                         "pip*"],
```

Comment:
Recommend against including the super generic process names `python` and `pip` because definition files join all values with `OR` statements so your query would turn out to be very false positive prone

(pseudo-code)
`process_name:python OR process_name:pip OR cmdline:mlflow OR domain:mlflow.org`

```suggestion
```